mozilla/fxa
# chore(multiple): update mocha wherever used

## Because

- Upgrading `mocha` in the profile server went painlessly, so we're updating `mocha` in all other packages that use it within the repo.

## This pull request

- Upgrades `mocha` from `^9.1.2` to `^10.0.0` in `browserid-verifier`, `fxa-auth-client`, `fxa-auth-server`, `fxa-content-server`, `fxa-geodb`, and `fxa-shared`. 

## Issue that this pull request solves

Closes: #13457 

## Checklist

_Put an `x` in the boxes that apply_


## To test
The way that I tested these changes is as follows:
I ran the test suite for each package before making any changes, for level setting. For all packages except for `fxa-content-server`, the tests are run entirely in the command line by running `yarn workspace <package-name-here> test`. For the `fxa-content-server`, you run that command and then also go to `localhost:3030/test/index.html` to see the outcomes of the unit tests. 
After noting how many tests pass for each package, I updated mocha (in the case of `fxa-shared` I also had to update a script), reinstalled for each package, rebuilt, and ran the tests again. The tests ran exactly as they had previously.
